src/show.jl

```diff
@@ -0,0 +1,224 @@
+function Base.show(io::IO, m::Model)
+    println(io, "MPSGE model with $(length(m._sectors)) sectors, $(length(m._commodities)) commodities and $(length(m._consumers)) consumers.")
+
+    if length(m._sectors) > 0
+        print(io, "  Sectors: ")
+        print(io, join(["$(s.name)" for s in m._sectors], ", "))
+        println(io)
+    end
+
```

Comment:
Removed benchmarks from printing when a model is printed. This caused over printing when an IndexedSector printed as an IndexedSector is potentially huge.